Jint/Native/Json/JsonSerializer.cs

```diff
@@ -547,22 +590,19 @@ public static PropertyEnumeration FromObjectInstance(ObjectInstance instance)
 
             private static void RemoveUnserializableProperties(ObjectInstance instance, List<JsValue> keys)
             {
-                for (var i = 0; i < keys.Count; i++)
+                for (var i = keys.Count - 1; i >= 0; i--)
```

Comment:
I even had to open a question about this as engines seem to differ: https://github.com/tc39/test262/issues/3811

Reply:
Oh damn, totally forgot that you can do that in JS. Damnit! Will undo the order change. It didn't contribute anything for the perf test anyway because the tests don't include any `undefined` - properties or non-plain objects.